Update track models for Season 2024-1

EstebanMz opened the pull request:
- 24 new tracks added: Aqueduct, Elementary 1, Elementary 2, Forest Mansion 2, HMS Invincible, Holiday Camp: California Edition, Hull Breach 3000 (RVA), Kadish Sprint, Library, MineZcraft, Motel in the Hood, Museum EX, Museum Unleashed (RVA), Paper Town 1, Port Limano 1, Ranch, Rigs Highway, RV Temple (RVA), Spring Visit, The Great Silence, Toys in the Hood 4, ToySoldierz, White Rose Chapel, Wladyslawowo.
- "Author" column added.

Note: Władysławowo should have this special 'ł' character but the web font used doesn't have it, so I'm writing it with a regular L.

EstebanMz committed:
Update tracks.csv for Season 2024-A

- 23 new tracks added
- Main Author column added
Note: Władysławowo should have this special 'ł' character but the web font used doesn't have it, so I'm writing it with a regular L.

EstebanMz committed:
Update rookie.csv for Season 2024-A

- IN: Beeble, Turbo Katto, Mr Bedtime, Toon Zoom, Cheat Code Central.
- OUT: HSF-1, Super Wheat, Gunior, Hot Spot, Bed Bug, Kanberra Kruiser, Iso-77A, El Gekko.

EstebanMz committed:
Add Avg. Lap Time column to tracks

- Added average_lap_time column to tracks.csv
- Minor typos fixed

EstebanMz committed:
Add Avg. Time Lap column to tracks

Sorry, I forgot to replace dot and commas with commas. Excel things after it exports.

BGMP commented:
Please split these changes into different PRs next time around. We've got cars and tracks stuff together in a single pull request now 😬

BGMP commented on models/cars/rookie.csv:
```diff
@@ -1,36 +1,33 @@
-Phat Slug,50.8,1.41,2.5,1.7,phat,TRUE
-Shiawase,41.6,1,5,2.0,shiawasecrimsonsolace,FALSE
-Creeki Leeki,49.9,2.16,1.2,3.0,gtcnd_creekileeki,FALSE
+name,speed,acc,weight,multiplier,folder_name,main_author,stock
```
Let's keep it singular, just `author`. Otherwise it implicitly feels like there're more authors for every creation.

```suggestion
name,speed,acc,weight,multiplier,folder_name,author,stock
```

EstebanMz replied:
Well, that's usually the case, there are few mods done from scratch by a single author

BGMP replied:
What about the ones which do not have more than 1 author?

EstebanMz replied:
Personally I think it's better to imply every mod was done by more than one person than the opposite. For example, a car could've been done by a single author, but that car is a remodel of another car done by a second author, which indirectly helped to make the car from the beginning.

BGMP replied:
Well, that's not the point I's trying to make here. I's just wondering how that is relevant to us or the site. We could instead add an ID row to construct a RVW URL on our end and just add a referencing button on the car pages.

For example, if we had https://www.revoltworld.net/dl/argon/ in the pack, the `rvw_id` column value would be `argon`for that car entry.

If your idea is giving as much credit to the authors as possible, I feel like doing what I'm proposing would be right thing to do. I did not do it earlier becuse there is no way I can assume the string that will complete RVW's URLs.

EstebanMz replied:
Can we check this suggestion about `rvw_id` later then? Also this suggest was done over `rookie.csv` which I already reverted to push it on another PR, so I guess I should press "Resolve conversation" button for now.

BGMP replied:
Yeah, let's review the RVW ID feature later. I'll create an issue for it

EstebanMz replied:
I've been checking that, just to find out that there are two ways the website generates these IDs.
The old way was with just the foldername, the current way takes the name of the mod, lowercase it, separates the words with '-' and removes any other special character.
I just noticed the existance of the old method after testing IDs in excel done with the new method.

BGMP replied:
I moved the conversation here: https://github.com/Re-Volt-America/RVA-Data/issues/15

BGMP commented on models/tracks/tracks.csv:
```diff
@@ -1,62 +1,81 @@
-Toytanic 2,T2,3,742,ship2,TRUE
-User-Deli: XL,DEL,3,390,user-delixl,FALSE
-Venice,VN,3,617,venice,FALSE
+name,short_name,difficulty,length,folder_name,main_author,stock,average_lap_time
```
```suggestion
name,short_name,difficulty,length,folder_name,author,stock,average_lap_time
```

EstebanMz committed:
Revert rookie.csv

Databases for cars will be handled in a separate PR.

EstebanMz committed:
Apply suggestions from code review

Co-authored-by: José Benavente <jose@bgm.cl>

EstebanMz committed:
Add last track to the pack & "(RVA)" tag

- IN: Hull Breach 3000 (RVA).
- Tracks exclusively modified for our pack now include "(RVA)" tag in their names.

EstebanMz committed:
Edit Toys in the Hood 4 (RVA version)
